test-jest/api/age-weight-key.spec.js

```diff
@@ -0,0 +1,41 @@
+const mockRequest = jest.fn()
+const mockRequestFileUpload = jest.fn()
+const AgeWeightKeyApi = require('../../src/api/age-weight-key')
+const Client = require('../../src/api/client')
+
+jest.mock('../../src/api/client', () => {
+  const originalModule = jest.requireActual('../../src/api/client')
+  return {
+    ...originalModule,
+    request: mockRequest,
+    requestFileUpload: mockRequestFileUpload
+  }
+})
+
+describe('age-weight-key', () => {
+  beforeEach(jest.clearAllMocks)
+
+  describe('getByYear', () => {
+    it.each([
+      ['1996'],
+      ['1990'],
+      ['2021']
+    ])('return Client.request', async (year) => {
+      mockRequest.mockImplementationOnce(() => ({ }))
+      await AgeWeightKeyApi.getByYear({}, year)
+      expect(mockRequest).toBeCalledWith(null, Client.method.GET, `reporting/reference/grilse-probabilities/${year}`)
```

Comment:
It's a small difference, but rather than checking the implementation, mock the return value of `Client.request` and check what you get back from AgeWeightKeyApi matches that.